pkgs/development/libraries/dirac/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "dirac-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/dirac/${name}.tar.gz";
+    sha256 = "1z803yzp17cj69wn11iyb13swqdd9xdzr58dsk6ghpr3ipqicsw1";
+  };
+
+  buildInputs = [ doxygen ];
+  enableParallelBuilding = true;
+
+  patches = [ ./dirac-1.0.2.patch ];
```

Comment:
It would be most likely possible to work around the warnings by putting a few extra flags into `NIX_CFLAGS_COMPILE`. That's what I do most of the time. On the other hand, this patch might be acceptable upstream...